packages/xo-server/src/xapi-object-to-xo.js

```diff
@@ -608,6 +608,7 @@ const TRANSFORMS = {
       snapshots: link(obj, 'snapshots'),
       tags: obj.tags,
       usage: +obj.physical_utilisation,
+      vdi_type: obj.type,
```

Comment:
camelCase

Reply:
Why didn't you use the `vdi.type`?

Reply:
I followed the existent implementations' notation. See https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-server/src/xapi-object-to-xo.js#L532

I can't use `vdi.type` see https://github.com/vatesfr/xen-orchestra/blob/0ed7ed7c7a9033397b6d13257e49b00f5f490667/packages/xo-server/src/xapi-object-to-xo.js#L601

Reply:
No, the existing implementation is `SR_type`, not `sr_type`.

Reply:
Then it should be `VDI_type` :+1: